pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -3092,6 +3092,25 @@ self: {
        license = stdenv.lib.licenses.mit;
      }) {};
 
+  "ChasingBottoms_1_3_1_9" = callPackage
```

Comment:
Can you try regenerating this file with the matching all-cabal-hashes version? That would make the PR much nicer to read and decrease the likelihood of rebase issues later on.

Reply:
Where can I find the instructions to do that ? It's not really clear to me sorry